### ISSUE TITLE
ISAR 3 is Ok for web or not?

### ISSUE THREAD

**Issue by astudilloalex** [isar/isar#1458]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Version**
 - Platform: iOS, Android, Mac, Windows, Linux, Web
 - Flutter version: [e.g. 1.5.4]
 - Isar version: [e.g. 0.5.0]

**Comment by nobkd** [comment]
No. Isar v3 does not support Web. Isav v4 does, but currently just for in-memory DB.
See https://github.com/isar/isar/issues/686#issuecomment-1660278982 and following comments.
Also see the general [Changelog for v4](https://github.com/isar/isar/blob/95e1f02c274bb4bb80f98c1a42ddf33f3690a50c/packages/isar/CHANGELOG.md#400-dev14)

**Comment by astudilloalex** [comment]
Oh!! i need save local data from offline work :| and after send to back.

**Comment by exaby73** [comment]
The current documentation is very misleading. Please update it and mention that Web is not supported. The [documentation](https://isar.dev/limitations.html#web-limitations) straight up says that it's supported though IndexedDB which is incorrect. This is a deal breaker for me. Even of Web had performance issues using IndexedDB, I'd still consider using it just for the type safe nature of it. I cannot use SQLite either since you can't access it on Android in a background thread which I need for my app. Isar was my best bet, and has disappointed because due to unclear documentation about web support, wasted hours of debugging and research only to find out from a GitHub issue more than a year old that web is not supported and won't be for a while considering v4 doesn't have persistence (in memory DB is kinda useless for most use cases). Love the effort, but need some more of that on the docs
